client/my-sites/email/email-management/email-home.tsx

```diff
@@ -173,14 +178,15 @@ const EmailHome = ( props: EmailManagementHomeProps ): ReactElement => {
 		);
 	}
 
-	if (
-		domainsWithEmail.length === 1 &&
-		domainsWithNoEmail.length === 0 &&
-		domainsWithEmail[ 0 ].domain === selectedSite?.domain &&
-		( domainsWithEmail[ 0 ].titanMailSubscription?.maximumMailboxCount ?? 0 ) > 0 &&
-		domainsWithEmail[ 0 ].titanMailSubscription?.numberOfMailboxes === 0
-	) {
-		page( emailManagementTitanSetUpMailbox( selectedSite?.slug, selectedSite?.domain ) );
+	if ( isSingleDomainThatHasEmail ) {
+		if (
+			( domainsWithEmail[ 0 ].titanMailSubscription?.maximumMailboxCount ?? 0 ) > 0 &&
+			domainsWithEmail[ 0 ].titanMailSubscription?.numberOfMailboxes === 0
+		) {
+			page( emailManagementTitanSetUpMailbox( selectedSite?.slug, selectedSite?.domain ) );
+		} else {
+			page( emailManagement( selectedSite?.slug, selectedSite.domain ) );
+		}
```

Comment:
```suggestion
		if (
			( domainsWithEmail[ 0 ].titanMailSubscription?.maximumMailboxCount ?? 0 ) > 0 &&
			domainsWithEmail[ 0 ].titanMailSubscription?.numberOfMailboxes === 0
		) {
			page( emailManagementTitanSetUpMailbox( selectedSite?.slug, selectedSite?.domain ) );
			return;
		}
		
		page( emailManagement( selectedSite?.slug, selectedSite.domain ) );
		
```

What do you think about simplifying like this?

Reply:
Readability-wise, I don't really mind either way. But I went ahead and changed it!